HISTORY.md

```diff
@@ -1,3 +1,8 @@
+unreleased
+===================
```

Comment:
You'll want to reduce the number of `=` to match the length of the header.

Reply:
Great catch! I fixed at d34bc21